### PR TITLE
Translate doc, built-in-makers.md, to Chinese

### PR DESCRIPTION
**Pull request by saiyan86**
Please comment & review!

### CODE REVIEW

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: [Configuring Metrics][configuring metrics] is not rendered correctly. please look at the bottom section of this file where I left a comment

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: [Druid aggregations][druid aggregations] is not rendered correctly, please look at the bottom section of this file where I left a comment

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: Fili talks to Druid. We are providing, in Fili, the same aggregation functionalities that Druid has. Given that, "且与下面表格中的操作一一对应" shall be made better by saying "和[Druid数据聚合功能][[druid aggregations]]一一对应".

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: This section, if you look at the actual rendering of this doc, is not showing. They are link references and are, hence, not to be translated

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: "Sketch集合操作" is a better translation than "集合操作"

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: “关于这些运算请参见” -> "这些运算符对应Druid的后期数据聚合操作"

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 👍 Beautiful translation!

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: We say it's constructed operation because some data aggregations are not simply done by Druid. One example is **average**. Average is a **Fili aggregator**, which is more complex than simply taking some summation. Could you come up with a better translation?

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: "单词"?

Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: "集合" could be better by saying something like "数据聚合"

Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: “限制Druid查询的规模” shall be improved.
A **grain** in Fili means the size of bucket we divide data into. For example, if grain is DAY and we are asking data from last week, we will get 7 result row; each row gives you the aggregated data in a day.

Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 数学表达 -> 进一步数学运算

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 允许我们在一个维度上进行集合操作 -> 可以在一个精度上聚合一个方面的数据

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 在这之前有些没翻译

File: translations/zh/built-in-makers-zh.md
Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 翻译成 -> 对应

Comment (QubitPi) [inline translations/zh/built-in-makers-zh.md]: 内层查询对每一个值为1的行加入一个常熟 -> 内层查询在每一个返回的结果列里插入一个值为1的常数列

**Review by ghost — CHANGES_REQUESTED** [review]
Overall, very nice translation! Make sure to add a link to the original doc that references to this translation, like that in [top level README](https://github.com/yahoo/fili/blob/master/README.md)

**Comment by michael-mclawhorn** [comment]
Is this still in progress?  Does someone need to adopt it?

**Comment by QubitPi** [comment]
@michael-mclawhorn I'm contacting the translator and ask him to update

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: I don't understand this. Could you please expand a little bit?

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: changed 限制 to 细分

Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: Thanks!

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed。

Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: which ones?

**Review by ghost — APPROVED** [review]

File: translations/zh/built-in-makers-zh.md
Comment (tarrantzhang) [inline translations/zh/built-in-makers-zh.md]:
```
下列建造器用于制造一级标准
```
可翻作
```
如下列举的是一些一级指标制造器，它们和Druid数据聚合功能一一对应。
```

File: translations/zh/built-in-makers-zh.md
Comment (tarrantzhang) [inline translations/zh/built-in-makers-zh.md]: post aggregators 可翻作“后聚合器”

File: translations/zh/built-in-makers-zh.md
Comment (tarrantzhang) [inline translations/zh/built-in-makers-zh.md]: 同上

File: translations/zh/built-in-makers-zh.md
Comment (tarrantzhang) [inline translations/zh/built-in-makers-zh.md]: 在我看来`Constructed Operations ` 这里指的是那些自定义的高复杂度运算。我认为可翻作“高复杂度运算”或“自定义运算”。

**Comment by QubitPi** [comment]
@tarrantzhang In order to let non-Chinese core team members to understand as much as possible, please use English whenever possible.

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: addressed

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: I included both Chinese and English version. I think this would be a better choice.

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: Yes! Constructed Operations -> 自定义 is the word I was looking for. Thanks!

File: docs/built-in-makers.md
Comment (tarrantzhang) [inline docs/built-in-makers.md]: Your editor may automatically changed this file (deleting padding spaces). We should avoid changing like this and keep this whole file untouched.

File: translations/zh/built-in-makers-zh.md
Comment (tarrantzhang) [inline translations/zh/built-in-makers-zh.md]: I'm not 100% sure if 
```which handle the same problem domain```
 in the original doc should be translated to 
```
因为二者功能重复
```

Comment (QubitPi) [reply translations/zh/built-in-makers-zh.md]: I think this looks OK.

File: docs/built-in-makers.md
Comment (QubitPi) [reply docs/built-in-makers.md]: Lines should not be added or deleted. Removing trailing spaces is fine, though.

File: translations/zh/built-in-makers-zh.md
Comment (saiyan86) [reply translations/zh/built-in-makers-zh.md]: I think this translation should be OK.

File: docs/built-in-makers.md
Comment (saiyan86) [reply docs/built-in-makers.md]: I can add those trailing spaces back if necessary.

**Review by tarrantzhang — APPROVED** [review]

**Comment by tarrantzhang** [comment]
lgtm :+1

**Comment by tarrantzhang** [comment]
Your PR should not include other people's commit. I suggest you roll back your branch to the point (e.g. 020605c) before you merge with other's. Then you should first syn your master with upstream,  and rebase your branch on top of it.

**Comment by QubitPi** [comment]
@tarrantzhang Fixed

**Pull request merged**
